_includes/about.html

```diff
@@ -2,8 +2,8 @@
     <div class="container">
         <div class="row">
             <div class="col-lg-12 text-center">
-                <h2 class="section-heading">Yup, that's right</h2>
-                <h3 class="section-subheading text-muted">We're the official community of hackers, builders, makers, and innovators at UMass Amherst</h3>
+                <h2 class="section-heading">Yup, that's right!</h2>
```

Comment:
I'd prefer a period over the exclamation point if we were gonna change it & I intentionally left out punctuation on the subheading part